asabiris/orchestration/sendemail.py

```diff
@@ -52,13 +52,15 @@ async def send_email(
 		:param attachments: a list of tuples, each tuple containing the filename and the file contents
 		"""
 
+		render_failed = False
```

Comment:
Raise the exception, this is not a good approach.

Reply:
@ateska I will refactor this spaghetti code.
It's in another MR